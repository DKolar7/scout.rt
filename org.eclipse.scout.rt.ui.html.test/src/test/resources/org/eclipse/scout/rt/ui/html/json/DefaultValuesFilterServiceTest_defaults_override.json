{
  "defaults": {
    "FormField": {
      "visible": false,
      "custom": "abc"
    }
  },
  "objectTypeHierarchy": {
    "Widget": {
      "Action": {
        "TableControl": null
      },
      "FormField": {
        "TableField": {
          "CustomTableField": null
        }
      }
    }
  }
}
