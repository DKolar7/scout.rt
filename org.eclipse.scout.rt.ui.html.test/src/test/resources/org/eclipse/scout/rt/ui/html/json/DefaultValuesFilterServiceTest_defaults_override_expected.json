{
  "defaults": {
    "FormField": {
      "enabled": true,
      "visible": false,
      "custom": "abc"
    }
  },
  "objectTypeHierarchy": {
    "Widget": {
      "Action": {
        "Menu": null,
        "TableControl": null
      },
      "FormField": {
        "TableField": {
          "CustomTableField": null
        }
      },
      "Tile": null
    }
  }
}
